core/dbt/cli/params.py

```diff
@@ -0,0 +1,386 @@
+    type=YAML(),
+)
+
+browser = click.option(
+    "--browser/--no-browser",
+    envvar=None,
+    help="Wether or not to open a local web browser after starting the server",
+    default=True,
+)
+
+cache_selected_only = click.option(
+    "--cache-selected-only/--no-cache-selected-only",
+    envvar="DBT_CACHE_SELECTED_ONLY",
+    help="Pre cache database objects relevant to selected resource only.",
+)
+
+compile_docs = click.option(
+    "--compile/--no-compile",
+    envvar=None,
+    help="Wether or not to run 'dbt compile' as part of docs generation",
+    default=True,
+)
+
+compile_parse = click.option(
+    "--compile/--no-compile",
+    envvar=None,
+    help="TODO: No help text currently available",
+    default=True,
+)
+
+config_dir = click.option(
+    "--config-dir",
+    envvar=None,
+    help="If specified, DBT will show path information for this project",
+    type=click.STRING,
+)
+
+debug = click.option(
+    "--debug/--no-debug",
+    "-d/ ",
+    envvar="DBT_DEBUG",
+    help="Display debug logging during dbt execution. Useful for debugging and making bug reports.",
+)
+
+# TODO:  The env var and name (reflected in flags) are corrections!
+# The original name was `DEFER_MODE` and used an env var called "DBT_DEFER_TO_STATE"
+# Both of which break existing naming conventions.
+# This will need to be fixed before use in the main codebase and communicated as a change to the community!
+defer = click.option(
+    "--defer/--no-defer",
+    envvar="DBT_DEFER",
+    help="If set, defer to the state variable for resolving unselected nodes.",
+)
+
+enable_legacy_logger = click.option(
+    "--enable-legacy-logger/--no-enable-legacy-logger",
+    envvar="DBT_ENABLE_LEGACY_LOGGER",
+    hidden=True,
+)
+
+event_buffer_size = click.option(
+    "--event-buffer-size",
+    envvar="DBT_EVENT_BUFFER_SIZE",
+    help="Sets the max number of events to buffer in EVENT_HISTORY.",
+    default=100000,
+    type=click.INT,
+)
+
+exclude = click.option("--exclude", envvar=None, help="Specify the nodes to exclude.")
+
+fail_fast = click.option(
+    "--fail-fast/--no-fail-fast",
+    "-x/ ",
+    envvar="DBT_FAIL_FAST",
+    help="Stop execution on first failure.",
+)
+
+full_refresh = click.option(
+    "--full-refresh",
+    envvar="DBT_FULL_REFRESH",
+    help="If specified, dbt will drop incremental models and fully-recalculate the incremental table from the model definition.",
+    is_flag=True,
+)
+
+indirect_selection = click.option(
+    "--indirect-selection",
+    envvar="DBT_INDIRECT_SELECTION",
+    help="Select all tests that are adjacent to selected resources, even if they those resources have been explicitly selected.",
+    type=click.Choice(["eager", "cautious"], case_sensitive=False),
+    default="eager",
+)
+
+#  TODO: Should this be exposed in the help text?
```

Comment:
I would just add a help text for consistency

Reply:
@emmyoop I believe you did the original work on this feature, got any thoughts on whether or not this needs to be shown in the dbt help text?

Reply:
@iknox-fa I'm unclear on why it would _not_ be included?  

Its purpose is to turn on verbose cache logging (for every event fired by `dbt/adapters/cache.py` I believe) if and only if the flag is turned on.

Reply:
Well, it's currently not included in the help text so `not included` = `parity`.  Looks like we want to add it so I'll go ahead and do so.